W-17027111-SF-Connector-Time-field-value-for-operations-KS (#2741)
mulesoft/docs-connectors: salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc

* <<upsert>>
* <<upsertMetadata>>

> 

Reviewer: move the note to before the list of operations

> If you set a `LocalDate` for a field in the payload, the input time gets automatically converted to `GMT` or the server’s timezone based on the `GregorianCalendar`. This conversion impacts date calculations or comparisons in Salesforce connector operations, as the `LocalDate` is adjusted to align with the server’s timezone or defaults to GMT.

Reviewer 2: ```suggestion
If you set a `LocalDate` for a field in the payload, the input time gets automatically converted to `GMT` or the server’s time zone based on the `GregorianCalendar`. This conversion impacts date calculations or comparisons in Salesforce connector operations, as the `LocalDate` is adjusted to align with the server’s time zone or defaults to GMT.
```

> If you set a `LocalDate` for a field in the payload, the input time gets automatically converted to `GMT` or the server’s time zone based on the `GregorianCalendar`. This conversion impacts date calculations or comparisons in Salesforce connector operations, as the `LocalDate` is adjusted to align with the server’s time zone or defaults to GMT.

Reviewer: ```suggestion
If you set `LocalDate` for a field in the payload, the input time is automatically converted to `GMT` or the server’s time zone based on the Gregorian calendar. This conversion impacts date calculations or comparisons in Salesforce Connector operations, as `LocalDate` is adjusted to align with the server’s time zone or defaults to GMT.
```

`queryParams: {AccountId : "a0w0E0000069MDxQAM" , ContactId: "0030E00000rRSJ6QAO" }` |  `#[payload]` |
| Apex Class Name a| String | Apex class name. |  | x

> | Apex Class Method Name a| String | Apex class method name. The method name format is like, 

Reviewer: ```suggestion
| Apex Class Method Name a| String | Apex class method name. The method name format is:
```

> `ApexMethodName&#94;RestResourceURL&#94;HttpMethod&#94;OutputType&#94;InputParameterName1=ParameterType1,InputParameterName2=ParameterType2`

Reviewer: add line break in between 2694 and 2695
